ingestion-beam/src/main/java/com/mozilla/telemetry/republisher/RepublishPerNamespace.java

```diff
@@ -37,13 +37,7 @@ public PDone expand(PCollection<PubsubMessage> input) {
     for (int i = 0; i < numDestinations; i++) {
       Destination destination = destinations.get(i);
       RepublisherOptions.Parsed opts = baseOptions.as(RepublisherOptions.Parsed.class);
-
-      // The destination pattern here must be compile-time due to a detail of Dataflow's
-      // streaming PubSub producer implementation; if that restriction is lifted in the future,
-      // this can become a runtime parameter and we can perform replacement via NestedValueProvider.
```

Comment:
I think we should preserve this comment

Reply:
This comment still exists in the per-doctype transform.